client/views/room/contextualBar/PruneMessages/PruneMessagesWithData.js

```diff
@@ -126,13 +126,13 @@ const PruneMessagesWithData = ({ rid, tabBar }) => {
 
 	const handleModal = () => {
 		setModal(
-			<DialogPruneMessages
+			<DeleteWarningModal
```

Comment:
Can you change to the GenericModal component? I think we should phase this component out...

Reply:
You rock buddy! I'll raise a PR replacing these modals to `GenericModal`